spmd/testing/devices.py

```diff
@@ -0,0 +1,35 @@
+from typing import TypeVar, cast
+
+import torch
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+
+from spmd import DeviceMesh
+
+DEVICE_TYPE = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
it would be great if we could just merge this logic to common_utils directly? don't see a reason to keep this file around once the decorator is resolved.

Reply:
Done